pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -202,6 +203,15 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			c.cephMgrOrchestratorModuleEnvs()...,
 		),
 		Resources: c.resources,
+		LivenessProbe: &v1.Probe{
+			Handler: v1.Handler{
+				HTTPGet: &v1.HTTPGetAction{
+					Path: "/",
+					Port: intstr.FromInt(metricsPort),
+				},
+			},
+			InitialDelaySeconds: 5,
```

Comment:
It can take a couple minutes to initialize the mgr modules. Should we increase the initial delay seconds?

Reply:
Like from when the MGR pod is started? I never experienced this to take so long. What would you suggest? 5 minutes?

Reply:
Configuring the mgr modules in nautilus seems to take 20-30 seconds from what i've seen. How about a delay of 60s to start the probe?

Reply:
ok it's `60` now